apps/e2e/a11y-storybook-e2e/src/e2e/skip-link.component.cy.ts

```diff
@@ -11,13 +11,9 @@ describe('a11y-storybook', () => {
           ),
       );
       it('should render the component', () => {
-        cy.get('#ready')
-          .should('exist')
-          .end()
-          .get('.sky-skip-link')
-          .should('exist')
-          .focus()
-          .should('be.visible');
+        cy.get('#ready').should('exist');
+        cy.get('.sky-skip-link').should('exist').focus();
+        cy.get('.sky-skip-link').should('be.visible');
```

Comment:
A new Cypress eslint rule recommends against chaining after certain interactions such as `click()`, `focus()`, and `screenshot()`.